seed/views/v3/data_quality.py

```diff
@@ -148,23 +148,22 @@ class DataQualityViews(viewsets.ViewSet):
             })
         }
     )
-    def create(self, request):
+    @api_endpoint_class
+    @ajax_request_class
+    @has_perm_class('requires_member')
+    @action(detail=True, methods=['POST'])
+    def start(self, request, pk):
```

Comment:
Since this breaks convention with the PK being Organization not DataQuality, what do you think about using `organization_pk` rather than just `pk` here?